web/apps/portal/src/styles/tailwind.css

```
@@ -0,0 +1,103 @@
+@import "tailwindcss";
+
+@source '../../../internal/ui/src/**/*.tsx';
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)